src/main/java/org/kairosdb/core/datastore/KairosDatastore.java

```diff
@@ -126,8 +126,10 @@ private void newCacheDirectory() {
 
 	@SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
could you also remove `ResultOfMethodCallIgnored` and add checks that all `File.delete` calls actually return true (`logger.error` otherwise)